contracts/modules/ModuleFactory.sol

```diff
@@ -28,6 +31,9 @@ contract ModuleFactory is IModuleFactory, Ownable {
 
     bool public isCostInPoly;
     uint256 public setupCost;
+    uint256 public usageCost;
+    uint256 public lastTimeUsageCostChange;
+    uint256 internal constant LOCKPERIOD = 24 hours;
```

Comment:
I am not sure how/why LOCKPERIOD is useful in its current form.

Perhaps we should do something like
1) user can change fee at any time.
2) new fee will become active only after lockperiod.

Reply:
Agreed - should be a two step propose:
  - factory owner proposes a usage fee change
  - factory owner can commit / finalise the change 24 hours after it has been proposed

Exception is when the module is first created of course.

Reply:
NB - if we did this for setup & usage costs, we could get rid of the logic in ST where the user has to pass an expected setup amount as well - not sure if this is worth fiddling with at this point.